docs-overrides/main.html

```diff
@@ -426,7 +420,7 @@ <h3 class="info-item__title">SedonaDB</h3>
               <h3 class="info-item__title">SedonaSpark</h3>
               <div class="info-item__description editor">Distributed batch processing on Apache Spark clusters.</div>
               <div class="info-item__cta">
-                <a href="#" class="btn-link">
+                <a href="https://sedona.apache.org/latest/setup/overview/" class="btn-link">
```

Comment:
Please don't use absolute paths, use relative path instead.

I believe it will look like "setup/overview/". Please try locally.